scripts/o3de/o3de/export_project.py

```diff
@@ -1180,9 +1180,9 @@ def register_setting(key: str, description: str, default: str = None, is_passwor
                                                             default='')
 
 SETTINGS_OPTION_BUILD_TOOLS       = register_setting(key='option.build.tools',
-                                                     description='The option to build O3DE toolchain executables. This will build AssetBundlerBatch and AssetProcessorBatch.',
+                                                     description='The option to build O3DE toolchain executables. This will build AssetBundlerBatch and AssetProcessorBatch. Unnecessary when using engine as SDK since tools are prebuilt.',
                                                      is_boolean=True,
-                                                     default='False')
+                                                     default='True')
```

Comment:
Why is this defaulting to `True` now? Lets leave it as it was

Reply:
This came from Gene's feedback earlier in the docs PR:
> Using the button now locally. Everything was very smooth.
> 
> Improvement suggestions (outside of documentation)
> 
>     1. I had to manually enable the "Build Toolchain" because I didn't have AssetProcessorBatch (most ppl wont have this)
> 
>     2. I had to manually select my seedlists
>        a. Would be nice automatically select all the seedlists inside of /AssetBundling/SeedList
>        b. The seedlist selection GUI only allows you to select one at a time (no multiselect)

I'm implementing his first suggestion based on his usage.